### PR TITLE
Remove xcolor

### PR DESCRIPTION
**Pull request by jwbos**
Turns out there was no dependency on xcolor at all. 
Various other minor typos corrected.

### CODE REVIEW

File: iacrcc/tests/test28/main.tex [hunk 1]
```diff
@@ -162,7 +162,7 @@ \section{Bibliography}
 \lipsum[1-20]
 
 % This sample uses bibtex rather than biblatex.
-\bibliography{template}
+% \bibliography{template}
```
Comment (kmccurley) [inline iacrcc/tests/test28/main.tex:165]: I'm not sure why this change was made, but it doesn't affect the test so it doesn't matter.

File: iacrcc/iacrcc.cls [hunk 2]
```diff
@@ -1,5 +1,5 @@
-\def\fileversion{0.34}
-\def\filedate{2023/07/28}
+\def\fileversion{0.35}
```
Comment (kmccurley) [inline iacrcc/iacrcc.cls:1]: we typically bump the version in iacrdoc.tex at the same time. Just make that change and submit.

**Review by kmccurley — APPROVED** [review]

File: iacrcc/tests/test28/main.tex (hunk 1 above)
Comment (jwbos) [reply iacrcc/tests/test28/main.tex:165]: On my end the template.bib file is not present in test28/ so I removed this.

File: iacrcc/iacrcc.cls (hunk 2 above)
Comment (jwbos) [reply iacrcc/iacrcc.cls:1]: Thanks, will do!
